misc/codegen/lib/schemadefs.py

```diff
@@ -295,6 +296,8 @@ def decorator(cls: type) -> _PropertyAnnotation:
             _ClassPragma(p, value=v)(annotated_cls)
         if replace_bases:
             annotated_cls.__bases__ = tuple(replace_bases.get(b, b) for b in annotated_cls.__bases__)
+        if add_bases:
+            annotated_cls.__bases__ = tuple(annotated_cls.__bases__) + tuple(add_bases)
```

Comment:
Equivalent and shorter:
```suggestion
            annotated_cls.__bases__ += tuple(add_bases)
```